main/attitude.h

```diff
@@ -64,7 +64,7 @@ Directional getAttitude(Directional gyro, bool hasLaunched){//only calibrated gy
   const int hz = 100; //number of readings per second
   static Directional oldChanges; //5-10 seconds ago
   static Directional lastChanges; //0-5 seconds ago
-  static int counter = 0;
+  static uint8_t counter = 0;
```

Comment:
the maximum value of a `uint8_t` is 255, but we need this number to be able to go up to 500. Thus we should use `uint16_t` instead

Reply:
should be fixed now. I noticed that `calibrateGyro` function had `uint8_t` for counter so I changed that to `uint16_t` as well.

Reply:
awesome nice catch